diff --git a/preset.js b/preset.js
new file mode 100644
index 0000000000000000000000000000000000000000..4bd63d3240021d58a8eb61d58a3b44183be247ae
--- /dev/null
+++ b/preset.js
@@ -0,0 +1 @@
+export * from './dist/preset.js';
diff --git a/preview.js b/preview.js
new file mode 100644
index 0000000000000000000000000000000000000000..542d45f8cf2617907e1d790c274acd38cdbd572e
--- /dev/null
+++ b/preview.js
@@ -0,0 +1 @@
+export * from './dist/preview.js';
